.travis.yml

```diff
@@ -3,9 +3,10 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-24.0.2
-    - android-24
+    - build-tools-26.0.2
+    - android-26
```

Comment:
Is there any reason why don't you use the newest libraries version, I mean: android-27, build-tools-27.0.0 and Gradle 4.3?